### PR TITLE
JAVA-30184: Changes made for Upgrade junit-jupiter library in the mai…

### PR DESCRIPTION
**Pull request by Bipinkumar27**
…n pom.xml

### CODE REVIEW

File: ksqldb/pom.xml [hunk 1]
```diff
@@ -45,6 +45,7 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <!--check this -->
```
Comment (alvin577austria) [inline ksqldb/pom.xml:48]: Can you check please if we can update this to the latest version.

File: libraries/pom.xml [hunk 2]
```diff
@@ -385,7 +385,7 @@
         <rcaller.version>3.0</rcaller.version>
         <rserve.version>1.8.1</rserve.version>
         <agrona.version>1.17.1</agrona.version>
-        <exec-maven-plugin.version>1.6.0</exec-maven-plugin.version>
+        <!--<exec-maven-plugin.version>1.6.0</exec-maven-plugin.version>-->
```
Comment (alvin577austria) [inline libraries/pom.xml:388]: Why do we need to comment this out?

File: microservices-modules/micronaut/pom.xml [hunk 3]
```diff
@@ -80,6 +80,7 @@
             <scope>compile</scope>
         </dependency>
         <dependency>
+            <!--check once -->
```
Comment (alvin577austria) [inline microservices-modules/micronaut/pom.xml:83]: Can you check please if we can update this to the latest version.

**Review by alvin577austria — CHANGES_REQUESTED** [review]

File: ksqldb/pom.xml (hunk 1 above)
Comment (Bipinkumar27) [reply ksqldb/pom.xml:48]: Updated

File: libraries/pom.xml (hunk 2 above)
Comment (Bipinkumar27) [reply libraries/pom.xml:388]: @alvin577austria the parent of this module already has the latest version of 3.1.0. I was just checking if that will work and if we are getting build errors if we use parent updated version.
I am removing this as this is working fine.

**Review by alvin577austria — APPROVED** [review]

**Review by alvin577austria — APPROVED** [review]

File: messaging-modules/spring-apache-camel/pom.xml [hunk 4]
```diff
@@ -77,7 +77,7 @@
     <properties>
         <camel.version>4.3.0</camel.version>
         <camel-swagger.version>3.22.0</camel-swagger.version>
-        <junit-jupiter.version>5.10.1</junit-jupiter.version> <!--Version not compatible with the one inherited from parent-->
+        <junit-jupiter.version>5.10.2</junit-jupiter.version> <!--Version not compatible with the one inherited from parent-->
```
Comment (alvin577austria) [inline messaging-modules/spring-apache-camel/pom.xml:80]: The parent module does not seem to have any junit-jupiter property set, can you remove the comment here please. Thanks!

**Review by alvin577austria — CHANGES_REQUESTED** [review]

**Review by alvin577austria — APPROVED** [review]

File: maven-modules/maven-exec-plugin/pom.xml [hunk 5]
```diff
@@ -84,6 +84,7 @@
     <properties>
         <maven-compiler-plugin.version>3.12.1</maven-compiler-plugin.version>
         <exec-maven-plugin.version>3.1.0</exec-maven-plugin.version>
+        <junit-jupiter.version>5.10.0</junit-jupiter.version>
```
Comment (alvin577austria) [inline maven-modules/maven-exec-plugin/pom.xml:87]: Is it necessary to use 5.10.0 here? is 5.10.2 not working?

**Review by alvin577austria — CHANGES_REQUESTED** [review]

File: maven-modules/maven-exec-plugin/pom.xml (hunk 5 above)
Comment (alvin577austria) [reply maven-modules/maven-exec-plugin/pom.xml:87]: I can't remember if I have asked about the same and you have already answered before though.

**Review by alvin577austria — APPROVED** [review]

**Pull request merged**